### DISCUSSION TITLE
Is Back-Channel Logout  supported?

### DISCUSSION THREAD

**Discussion by rickwang7712** [zitadel/oidc#621] [Q&A] [answered]
Hi, 
I saw there is a commit for front channel logout, but I'm wondering if back channel logout is also supported?
This is useful for implementing single sign out for all RPs.

spec: https://openid.net/specs/openid-connect-backchannel-1_0.html

Thanks!

**Comment by muhlemmer** [comment]
No, we currently do not support back-channel logout. If there is lots of popular or commercial demand, we will consider implementing it.

**Comment by rickwang7712** [comment]
Hi @muhlemmer ,
Thanks for your reply!

May I ask is it possible to achieve single sign out with this library?
Thanks!

**Reply by muhlemmer** [reply]
Yes, using RP-initiated logout.

**Comment by Congee** [comment]
The ability to run actions on a logout event would be a great workaround https://github.com/zitadel/zitadel/issues/7246

**Reply by hifabienne** [reply]
We have just added back channel logout recently: https://github.com/zitadel/oidc/pull/671

**Comment by hifabienne** [comment, marked as answer]
Back channel logout is now supported  https://github.com/zitadel/oidc/pull/671
